crates/router/src/configs/settings.rs

```diff
@@ -153,6 +153,7 @@ pub struct Connectors {
 
     // Keep this field separate from the remaining fields
     pub supported: SupportedConnectors,
+	pub forte: ConnectorParams,
```

Comment:
run `cargo clippy --all-features --all-targets && cargo +nightly fmt --all` to fix formatting issues

Reply:
done